pkg/api/orchestrators.go

```diff
@@ -181,6 +181,9 @@ func kubernetesUpgrades(csOrch *OrchestratorProfile) ([]*OrchestratorProfile, er
 	case currentMajor == 1 && currentMinor == 7:
 		nextMajor = 1
 		nextMinor = 8
+	case currentMajor == 1 && currentMinor == 8:
+		nextMajor = 1
```

Comment:
shouldn't this be a seperate PR as this feels unrelated and should be easy to push through on it's own.

Reply:
agreed